index.d.ts

```diff
@@ -6,7 +6,7 @@ export interface ServiceConfiguration {
 }
 
 export type BaseAuthConfiguration =
-  | {
```

Comment:
I'm pretty sure this was a style choice that @jevakallio introduced using an automated prettier format. Have you accidentally skipped our lint-staged formatting? https://github.com/FormidableLabs/react-native-app-auth/blob/master/package.json#L18-L19

Reply:
Ah ok, I see. Never seen it written like that, it reads like "nothing or this or that" to me so I was afraid if it wouldn't accept nothing the way it was. Hope that makes sense :D Anyways, I've reverted it. Rest should be good to go in.